gulpfile.js

```diff
@@ -110,13 +110,20 @@ function createWebpackConfig(defines, output) {
     module: {
       loaders: [
         {
+          test: /(\bweb\b).*?\.js$/,
```

Comment:
Unless I'm totally misunderstanding the RegExp, won't this limit Babel to files in the `/web` directory? Don't we also want to allow ES6 in files in the `/src` directory too?